pkg/gateway/nginx.go

```diff
@@ -143,12 +151,18 @@ func renderConfig(cfg *NGINXConfig, sm *ServiceMap) ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
-func newLoggingNGINXManager() NGINXManager {
-	return &loggingNGINXManager{status: nginxStatusStopped}
+func newLoggingNGINXManager(hp int) NGINXManager {
+	manager := loggingNGINXManager{
+		status: nginxStatusStopped,
+		cfg:    newNGINXConfig(hp),
+	}
+	log.Printf("created newLoggingNGINXManager: %+v", manager)
+	return &manager
 }
 
 type loggingNGINXManager struct {
 	status string
+	cfg    NGINXConfig
```

Comment:
Potentially something that would be better served with unit tests (which I'll hopefully get to tomorrow). I don't think it's a bad idea to continue to pretend that the logging manager has a config when it's live cousin does.

Are you in favor of `cfg` from this struct if I add the logging outside of `NewLoggingNGINXManager` factory?